.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Gingersnap db-syncer - Build and publish
+
+on: 
+  push:
+    tags:
+      - 'v*'
+      
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}    
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Configure Git
+        run: |
+          git config --global user.name "infinispan-qe-bot"
+          git config --global user.email "q*@infinispan.org"
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with: 
+          token: ${{ secrets.MY_TOKEN }}  
+          fetch-depth: 0
+
+      - name: Checkout latest tag
+        id: vars
+        run: | 
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
Shouldn't the tag being pushed inform what is used here?

I think doing:

```yaml
      - name: Checkout
        uses: actions/checkout@v2
```

On a tag push is sufficient for checking out the relevant tag.

Reply:
Corrected.